client/src/components/Feed/Deals/DealItem/DealItem.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import "./DealItem.css";
+
+class DealItem extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      dealItem: []
+    };
+  }
+
+  async componentDidMount () {
+    //return the param value
+    const { deal_name } = this.props.match.params;
```

Comment:
@simonnguyen3054 Can you explain the line `const { deal_name } = this.props.match.params` ?

Reply:
This is just object destructuring @avakrishn 

Anytime now when I use deal_name variable, I'm actually passing the value. So in this case, 
```const dealItemArr = await fetch(`http://localhost:3001/api/deals/${deal_name}`); ```

${deal_name} is the value